From 3cab2afc418f3363152708ab8bfaeb5380fa1385 Mon Sep 17 00:00:00 2001
From: Luo Chunsheng <luochunsheng@huawei.com>
Date: Tue, 16 Apr 2019 05:02:46 -0400
Subject: [PATCH] Currently we have this test case: 	grubDisplayTest grub.1
 defaultkernel/g.1 --default-kernel when no boot partition, it outputs without
 "/boot" Prefix, then don't match defaultkernel/g.1, so add test option
 "--boot-filesystem=/boot" to fix make test fail.

---
 test.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test.sh b/test.sh
index 5d59216..6896985 100755
--- a/test.sh
+++ b/test.sh
@@ -299,7 +299,7 @@ grubDisplayTest grub.10 defaulttitle/g.10 --default-title
 grubDisplayTest grub.11 defaulttitle/g.11 --default-title
 
 testing="GRUB display default kernel"
-grubDisplayTest grub.1 defaultkernel/g.1 --default-kernel
+grubDisplayTest grub.1 defaultkernel/g.1 --boot-filesystem=/boot --default-kernel
 
 testing="LILO default directive"
 liloTest lilo.1 default/l1.1 --set-default=/boot/vmlinuz-2.4.18-4
-- 
1.8.3.1

